collect_app/src/main/java/org/odk/collect/android/formmanagement/ServerFormDownloader.java

```diff
@@ -216,9 +216,16 @@ boolean installEverything(String tempMediaPath, FileResult fileResult, Map<Strin
             UriResult uriResult = null;
 
             try {
-                // Copy form to forms dir
-                File formFile = new File(formsDirPath, fileResult.file.getName());
-                FileUtils.copyFile(fileResult.file, formFile);
+                File formFile;
+
+                if (fileResult.isNew()) {
```

Comment:
For now, only copy the new form file if it's new to prevent anything weird happening when we copy a file into itself. I think we probably want to fix all the weirdness with the form file being new or not but it seems like it would really require ripping out any direct usage of the content provider (and switch to our newer repos) to make anything make sense.

As this is presumably going to be a hot fix and will have to be cherry-picked back onto master (which will involve some nasty merging) I just went with a quick hack solution.